multicluster/test/integration/resourceexport_controller_test.go

```diff
@@ -0,0 +1,262 @@
+/*
```

Comment:
Could we change to "//" for comments to be consistent with other files. In general, "//" is recommended for Golang.

Reply:
done

Reply:
Forgot to push the change?

Reply:
sorry, pushed the latest change now, please check. thanks.